package.json

```diff
@@ -8,11 +8,10 @@
     "build": "./bin/build-code",
     "watch": "./bin/build-code -dw",
     "build:test": "./bin/build-tests",
-    "build:test:watch": "./bin/build-tests -dw",
     "prepublish": "npm run build",
     "clean": "rm -rf lib",
     "test": "npm run build:test && npm run build && karma start --single-run",
-    "test:watch": "npm run build:test && npm run build && concurrently 'npm run build:test:watch' 'npm run watch' 'karma start'",
+    "test:debug": "npm run build:test && npm run build && karma start",
```

Comment:
Getting all the watchers set up for systemjs, webpack, and karma was too much of a hassle so now I'm just starting up karma without --single-run and calling that "debug".